libcloudforensics/aws.py

```diff
@@ -854,12 +1065,56 @@ def CreateVolumeCopy(zone,
 
   except RuntimeError as exception:
     error_msg = 'Copying volume {0:s}: {1!s}'.format(
-        volume_id, exception)
+        (volume_id or instance_id), exception)
     raise RuntimeError(error_msg)
 
   return new_volume
 
 
+def StartAnalysisVm(vm_name,
+                    default_availability_zone,
+                    boot_volume_size,
+                    cpu_cores=4,
+                    ami=UBUNTU_1804_AMI,
+                    attach_volume=None,
+                    device_name=None,
+                    dst_account=None):
+  """Start a virtual machine for analysis purposes.
+
```

Comment:
Please add a sentence or two explaining what happens if the VM already exists, if the VM exists but is turned off (is that a thing in AWS?) or if the VM doesn't exist.